pegasus/src/forms.rb

```diff
@@ -1,34 +1,8 @@
-require 'cdo/poste'
 require 'cdo/regexp'
 require src_dir 'database'
+require File.join(CDO.root_dir, 'lib/forms/pegasus_form_errors')
```

Comment:
Our usual pattern is `require_relative`. Is there a reason to prefer this over this? Note that I think the above is cleaner.